implementation/test/TBTCSystemTest.js

```diff
@@ -103,7 +103,9 @@ describe("TBTCSystem", async function() {
       await increaseTime(totalDelay.toNumber() - expectedRemaining)
 
       const remaining = await tbtcSystem.geRemainingSignerFeeDivisorUpdateTime.call()
-      expect(remaining).to.eq.BN(expectedRemaining)
+      expect([expectedRemaining, expectedRemaining + 1]).to.include.toString(
```

Comment:
Do we need to check the range here? Why won't we get specific `expectedRemaining` value as a result?

Reply:
It's just a way to avoid potentially non-deterministic tests. If a second is incremented locally before the end of the computation/promise resolution there will be an unexpected block timestamp increase.